pkg/debug/checkups/logs.go

```diff
@@ -0,0 +1,80 @@
+package checkups
+
+import (
+	"archive/zip"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/kolide/launcher/pkg/agent/types"
+)
+
+type Logs struct {
+	k       types.Knapsack
+	status  Status
+	summary string
+}
+
+func (c *Logs) Name() string {
+	return "Logs"
+}
+
+func (c *Logs) Run(_ context.Context, fullFH io.Writer) error {
+
+	debugLog := filepath.Join(c.k.RootDirectory(), "debug.json")
+
+	stat, err := os.Stat(debugLog)
+	if err != nil {
+		c.status = Failing
+		c.summary = fmt.Sprintf("failed to stat debug log: %s", err)
+		return nil
```

Comment:
Because it represents something like no logs, or no permission, which are detectable as data failures. I think most checkups won't have an error state. Hopefully other comment updates help clarify.